video/randominput.h

```diff
@@ -65,11 +66,11 @@ class RandomInput : public Input {
     cv::Size m_Size;
     std::string m_CameraName;
     GeneratorType m_Generator;
-    std::uniform_int_distribution<uchar> m_Distribution;
+    std::uniform_int_distribution<int> m_Distribution;
```

Comment:
What was wrong with ``std::uniform_distribution<uchar>``?

Reply:
MSVC won't let you have a ``std::uniform_distribution<uchar>`` and says it violates some standard or other (which is a bit odd because it seems useful)

Reply:
How irritating - https://en.cppreference.com/w/cpp/numeric/random/uniform_int_distribution does agree that uchar is within the dread realm of undefined behaviour